### PR TITLE
Milestones: Add structure, style and functionality for to-do list

### PR DESCRIPTION
**Pull request by geekelo**
Things worked on;
- Setup webpack
- Add HTML structure for webapp
- Add CSS style rules for webapp
- Add dynamic rendering of tasks from Javascript
- Use object-oriented programming in Javascript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,7 +118,7 @@ Head to the <a href="#live-demo">LIVE DEMO SECTION</a> for Live links and presen
 
 > View the live demo of this project by clicking the text below.
 
-- [See Live Demo](https://geekelo.github.io/awesome-books-webapp/)
+- [See Live Demo](https://geekelo.github.io/oti-task-organizer-webapp/dist/index.html)
 <!-- - [First Presenation Video](https://www.loom.com/share/719c333faa524c5db6930878f33b809b) -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Ismailco) [inline README.md:124]: - [ ] The LICENSE link is broken, kindly make sure to fix it so your README links are working properly

File: src/print.js [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+        completed: false,
+        description: 'Setup webpack',
+      },
+      {
+        id: 1,
+        completed: false,
+        description: 'Set up webpack dev-server',
+      },
+      {
+        id: 1,
```
Comment (Ismailco) [inline src/print.js]: - [ ] Kindly make sure to change the indexes of the second and third tasks, it should be `2` then `3`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @geekelo, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Clean code
✔️ Nice UI
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @geekelo 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Clean code
✔️ Used GitHub Flow correctly

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**